### PR TITLE
STL-1379 Add back button to stdmin & adjust text

### PR DESCRIPTION
**Pull request by SannyNguyenHung**
# Short Description
- Add back buttton to the stdmin page
- Add dot to text

### CODE REVIEW

File: webapp/app/forms/steps/lotse/steuerminderungen.py [hunk 1]
```diff
@@ -59,9 +59,10 @@ def render(self, **kwargs):
             form={
                 'action': self.render_info.submit_url,
                 'csrf_token': generate_csrf(),
-                'show_overview_button': bool(self.render_info.overview_url),
+                'show_overview_button': bool(self.render_info.overview_url)
```
Comment (JulianRoesner) [inline webapp/app/forms/steps/lotse/steuerminderungen.py:62]: Out of curiosity: why did you delete the ','?

File: webapp/app/model/components/__init__.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ class DeclarationIncomesProps(StepFormProps):
 
 
 class SelectStmindProps(StepFormProps):
-    pass
+    prev_url: str
```
Comment (JulianRoesner) [inline webapp/app/model/components/__init__.py:59]: I am wondering if we should put this somewhere more general like into a `LotseStepFormProps`. But this should not be part of this PR, but maybe we add a TODO comment?

```suggestion
    prev_url: str  # TODO place this at a more general place when implementing more StepFormProps for the LotseFlow
```

**Review by JulianRoesner — APPROVED** [review]
LGTM. Only small questions and remarks

File: webapp/app/forms/steps/lotse/steuerminderungen.py (hunk 1 above)
Comment (SannyNguyenHung) [reply webapp/app/forms/steps/lotse/steuerminderungen.py:62]: It just annoyed me because it is the last element... :sweat_smile:

File: webapp/app/model/components/__init__.py [hunk 3]
```diff
@@ -56,7 +56,7 @@ class DeclarationIncomesProps(StepFormProps):
 
 
 class SelectStmindProps(StepFormProps):
-    pass
+    prev_url: str  # TODO place this at a more general place when implementing more StepFormProps for the LotseFlow
```
Comment (frederike-ramin) [inline webapp/app/model/components/__init__.py:59]: Why aren't we doing this right away? I think it actually belongs to the `StepFormProps`.
